.buildkite/core.rayci.yml

```diff
@@ -67,7 +67,7 @@ steps:
         --parallelism-per-worker 3
       # doc examples
       - bazel run //ci/ray_ci:test_in_docker -- //doc/... core
-        --except-tags gpu,post_wheel_build,doctest
+        --except-tags gpu,post_wheel_build,xcommit,doctest
```

Comment:
What's the `xcommit` tag represent?

Reply:
It indicates that the test should run on civ1 infra instead of civ2